libraries/AP_Relay/AP_Relay.cpp

```diff
@@ -229,6 +209,46 @@ bool AP_Relay::send_relay_status(const GCS_MAVLINK &link) const
 }
 #endif  // AP_MAVLINK_MSG_RELAY_STATUS_ENABLED
 
+void AP_Relay::update()
+{
+    for (uint8_t instance = 0; instance < AP_RELAY_NUM_RELAYS; instance++) {
+        const int8_t pin = _params[instance].pin;
+        if (pin == -1) {
+            // no valid pin to set it on, skip it
+            continue;
+        }
+
+        const AP_Relay_Params::Function function = _params[instance].function;
+        if (function <= AP_Relay_Params::Function::none || function >= AP_Relay_Params::Function::num_functions) {
+            // invalid function, skip it
```

Comment:
We could set to the default stat in this case. Although it probably doesn't make much difference.

Reply:
I think we should leave it alone. It's both less code,and  it doesn't hurt anything, and it's actually more friendly for version hopping where the channel might have a use in newer builds but you went back to an old build to test something.